src/OrchardCore.Cms.Web/NLog.config

```diff
@@ -26,6 +26,8 @@
         <!-- all warnings and above go to the file target -->
         <logger name="*" minlevel="Warn" writeTo="file" />
 
+        <logger name="*" minlevel="Info" writeTo="console" />
```

Comment:
not sure we really wanna do this by default ?

Reply:
Hehe i had the same comment from this morning but did not submit my review :)

Reply:
Whoops, this is a line I've added for debugging.